settings.gradle.kts

```diff
@@ -0,0 +1 @@
+rootProject.name = "bookmarks"
```

Comment:
It worth adding new line at the end of each file. Otherwise Github would show you this beautiful stop sign.

Reply:
Fair, I agree. I'll fix it. But anyway, why doesn't IDEA add new lines when creating Kotlin files? :(

Reply:
Fixed